tsconfig.json

```diff
@@ -8,7 +8,8 @@
     "sourceMap": true,
     "declaration": true,
     "importHelpers": true,
-    "strict": true
+    "strict": true,
+    "skipLibCheck": true
```

Comment:
needed to stop `typescript` from throwing errors in the node_module gitbeaker package as it seems to use newer Types for string literals